a3p-integration/proposals/a:upgrade-14/ante-fees.test.js

```diff
@@ -1,74 +0,0 @@
-import test from 'ava';
-
-import { CHAINID, GOV1ADDR, GOV2ADDR, agd } from '@agoric/synthetic-chain';
-
-test(`ante handler sends fee only to vbank/reserve`, async t => {
```

Comment:
we lose this test? that seems unfortunate

Reply:
it seems a shame to go backwards w.r.t.

 - https://github.com/Agoric/agoric-sdk/issues/8961

Reply:
so it looks like last time (#8755) we were content to have the tests in agoric-3-proposals: https://github.com/Agoric/agoric-3-proposals/blob/main/proposals/71%3Aupgrade-14/ante-fees.test.js

But that only tests that the ante handler works in -14.
This feature is still relevant in 15, so it would be nice to keep it.

Reply:
After an upgrade the tests move to agoric-3-proposal repo. In particular https://github.com/Agoric/agoric-3-proposals/blob/main/proposals/71%3Aupgrade-14/ante-fees.test.js

Acceptance tests are somewhat orthogonal